### ISSUE TITLE
Empty vector output resultant of training using given training data

### ISSUE THREAD

**Issue by Amuqeet09** [NinV/Korean-License-Plate-Recognition#5]
Thank you for the code. 

I successfully run your all codes. But when I train it with your given data, it returns empty vector. What could be the possible problem? 

Is it possible to share KarPlate dataset? If possible, please share about the number of training/testing data.

**Comment by jpsalada** [comment]
same issue here

**Comment by NinV** [comment]
I think the problem is with the default training parameters. You can experiment with different learning rates, batch size, and the number of epochs. The training loss should be low enough for the model to show some acceptable results. You can use the code in "evaluate.py" file to check the loss value of the pre-trained model and use it as a reference for tuning training parameters.

**Comment by AbdulMoqeet** [comment]
Thank you for the reply. I found that the test loss was not great which means training was not that good. Eventually, I used your pretrained model and fine tune it to work on it. 

But, I have to retrain for my scenario.

**Comment by jpsalada** [comment]
@NinV thank you so much for replying. I will try that. Would it be possible the training configuration parameters that you have used to create your model?

**Comment by NinV** [comment]
The pretrained model is trained using this command:

`python train.py -l data/label.json -i data/train_images --valid_label data/test.json --valid_img_dir data/test_images --save_weights_only --load_all --learning_rate 0.001 --decay_steps 10000 --decay_rate 0.995 --num_steps 100000 --batch_size 32`

**Comment by Amuqeet09** [comment]
I came across that network was not fully trained that was causing resultant empty vector.
